cost/aws/object_storage_optimization/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v3.2
+
+- Added a filter to the S3 Objects that will be moved to deeper or glacier storage class.
```

Comment:
I'm not clear on what this means. Could you elaborate?

Reply:
The description does not reflect the essence of the changes. This filter was already present, but filtering took place after receiving tags. Now we filter filter and then receive tags.

Reply:
So if I understand correctly, the policy's functionality did not change, but we rearchitected it a bit so that it will be more efficient. In that case, this line should be something like:

- Optimized policy code to speed up execution

The key is to remember that the CHANGELOG is going to be read by ordinary users who might have no knowledge or awareness of code, programming, etc. They need the bottom line of how the changes impact them; in this case, we're making the policy execute more quickly.

Reply:
Thank you Shawn, have a great weekend